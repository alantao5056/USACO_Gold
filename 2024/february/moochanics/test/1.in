4
2
1 11
1 1
2
1 12
1 1
2
1 11
4 6
2
1 11
4 5
