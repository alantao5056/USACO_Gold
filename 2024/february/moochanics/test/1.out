9 9
11 11
1 1
3 3
